lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -147,30 +143,34 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             )
 
         host = self.platform_runbook.hosts[0]
+        host_node: Node
         if host.is_remote():
             assert host.address
             if not host.username:
                 raise LisaException("Username must be provided for remote host")
             if not host.private_key_file:
                 raise LisaException("Private key file must be provided for remote host")
 
-            self.host_node = RemoteNode(
+            self._is_host_remote = True
```

Comment:
nit: It would be good to set `self.host_node_object` and `self._is_host_remote` together.